plugins/source/hubspot/.github/workflows/release.yaml

```diff
@@ -0,0 +1,38 @@
+name: release
```

Comment:
Is this file needed?

Reply:
No idea what that is :)

Reply:
Did you use the scaffold CLI for the plugin? If so, it comes from https://github.com/cloudquery/cloudquery/tree/d75df29d468a84724229ecef6ce558d31e665707/scaffold/cmd/templates/source/.github/workflows, but it is not needed for the monorepo, so we can delete it